process.go

```diff
@@ -0,0 +1,159 @@
+/*
+ * File: process.go
+ *
+ * Description: functions for handling libcontainer processes
+ */
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/opencontainers/runc/libcontainer"
+	_ "github.com/opencontainers/runc/libcontainer/nsenter"
+)
+
+// The process definition.
+type Process struct {
+
+	// Id generated via lcluster
+	uuid int64
+
+	// Refs to the libcontainer object
+	container libcontainer.BaseContainer
+	proc      *libcontainer.Process
+}
+
+//! Function to initialize the libcontiner process handling.
+/*
+ * @return    none
+ */
+func init() {
+
+	//
+	// NOTE: this function was adapted from the etcd docs, ergo whence
```

Comment:
Ah, this could use more precise clarification, thanks for noticing that.

https://github.com/rbisewski/lclusterd/commit/77adddf7de69b73b3a27828d509f2046ffdc6ecd